tests/drivers/adc/adc_dma/boards/frdm_k82f.overlay

```diff
@@ -10,3 +10,9 @@
 	high-speed;
 	periodic-trigger;
 };
+
+&edma0 {
+        dma-buf-alignment = <32>;
```

Comment:
same as above

Reply:
@teburd  can you help to review and approve? Thanks